docs/content/docs/recipes/temporary-storage.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Using Fluid for temporary storage
+menuPosition: 3
+---
+
+Some developers may wish to use the Fluid container for temporary storage only, choosing to use another data store for the long-term storage of the data after the collaboration period has passed -- or not storing the data at all, as it may have no usefulness beyond the collaboration period.
+
+In this scenario, we assume the data in the primary data store will be temporarily copied to a Fluid container so that it can participate in a real-time collaboration experience for some period of time, and that it will not be edited in the original location while it's in the Fluid container. All of the business logic described here is assumed to be executed on the client in the context of the current user.
+
+![high level diagram of data import and export](https://fluidframework.blob.core.windows.net/static/images/import-export.jpg)
+
+## Getting data into the container
+
+The application developer is responsible for connecting to the external data store from their client-side web application, finding the appropriate data, and marshalling the data into a new Fluid container. We recommend creating the container in a detached state, inserting the external data into new Fluid objects, and then attaching the container. This avoids the situation where another user opens the container and the seed data is not yet present.
```

Comment:
I'm not familiar with the distinction between creating the container in a detached state and an attached state. If this is documented somewhere, we should link to that documentation here. If it isn't, we should document it.